components/aggregation/mat/Form.js

```diff
@@ -148,24 +152,28 @@ export const Form = ({ onSubmit, testNames, query }) => {
           />
         </Box>
       </Flex>
-      <Flex justifyContent='space-between'  alignItems='center'>
-        <Box>
+      <Flex my={2} justifyContent='space-between' sx={{
+        '@media screen and (max-width: 476px)': {
+          flexDirection: 'column',
+        }
+      }}>
+        <Box width={[1, 0.18]}>
           <StyledLabel>
             Test Name
           </StyledLabel>
           <Controller
             name='test_name'
             control={control}
             render={({field}) => (
-              <Select {...field}>
+              <Select {...field} sx={{ width: '100%' }}>
```

Comment:
Same as the earlier comment about using object literal as prop. Also applies to other such instances below.